keymapviz/legends.py

```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
+import configparser
 
 legends = {
```

Comment:
How about moving these to the default config file too? It would allow the user to disable or override them as they like.

Reply:
The user should be able to override them by adding them to a provided config file

Reply:
That's true, although there is a bit of a discoverability issue since user needs to find this code to figure out there is legend replacement happening without the config file.

Reply:
Either way is fine by me. Not sure how @yskoht feels about this? Not sure if he gets notified about comments here though. Maybe it would be better to address an issue if you feel like this should be changed :-)

Reply:
@caksoylar @daanhorn 

Thanks for comments!

> How about moving these to the default config file too?

I agree your idea. I believe that `keymapviz/legends.py` is no longer needed because of #27.
I'm thinking where to put the default config file. `~/.keymapviz` or somewhere in keymapviz package.

Reply:
I think a dotfile might make sense, either `~/.keymapviz` or `~/.config/keymapviz/config.properties` (with `~/.config` overridden by `$XDG_CONFIG_HOME` if defined). If it is in the package it might be hard to find when the package is installed with `pip`.

Reply:
Dotfile would be a nice solution indeed!